command/volume.go

```diff
@@ -0,0 +1,46 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type VolumeCommand struct {
+	Meta
+}
+
+func (c *VolumeCommand) Help() string {
+	helpText := `
+Usage: nomad volume <subcommand> [options]
+
+  volume groups commands that interact with volumes.
+
+  Register a new volume or update an existing volume:
+
+      $ nomad volume register <input>
+
+  Examine the status of a volume:
+
+      $ nomad volume status <id>
+
+  Deregister a volume, removing it from the system:
```

Comment:
I had this comment on the first pass at the PR but it looks like it might have gotten dropped or made outdated accidentally (it wasn't marked as resolved):

> "Removing it from the system" sounds like it does a lot more than it does. Maybe just cut it down to "Deregister an unused volume"?

Reply:
Good call, 55db2fa3a